targets/psa_switch/psa_switch.cpp

```diff
@@ -149,13 +150,14 @@ PsaSwitch::receive_(port_t port_num, const char *buffer, int len) {
 
   // setting standard metadata
 
-  phv->get_field("standard_metadata.ingress_port").set(port_num);
+  phv->get_field("psa_ingress_parser_input_metadata.ingress_port").set(port_num);
+  phv->get_field("psa_ingress_parser_input_metadata.packet_path").set(PKT_INSTANCE_TYPE_NORMAL);
```

Comment:
I thought psa.p4 introduced an enum for this, why not use the appropriate enum member value that should be included in the JSON?

Reply:
Actually that's a great point. There has been an ongoing issue with enum translations in p4c so they currently aren't emitted in the JSON. However PSA_PacketPath_t is correctly translated just not emitted.

I can make changes to p4c, have those committed, and come back to this PR and have logic changed to use the enum member values in the JSON.

Reply:
Since "normal" packet type will always be 0, you can just put a TODO as a remainder, then come back to it later. There is no need to delay merging this PR for this.